test/lib/ufe/testComboCmd.py

```diff
@@ -805,7 +805,7 @@ def testFallback(self):
         checkPivotsAndCompensations(self, mayaObj, usdSphere3d)
         checkPivotsAndCompensations(self, mayaObj, usdFallbackSphere3d)
 
-    @unittest.skipIf(mayaUtils.previewReleaseVersion() < 124, 'Center pivot command is only available in Maya Preview Release 124 or later.')
+    @unittest.skipIf(int(cmds.about(apiVersion=True)) <= 20220000, 'Center pivot command is only available in Maya Preview Release 124 or later.')
```

Comment:
Bit awkward that the reason for the skip doesn't match the conditional

Reply:
Agreed. @HamedSabri-adsk Please adapt the message.